File: sqlInsert.py
#GITHUB Insert into a SQL database.
import MySQLdb

db = MySQLdb.connect(host="localhost", user="logcal20", passwd="password", db="school")
cur = db.cursor(MySQLdb.cursors.DictCursor)
 
name = input("Please enter student's name. ")
age = input("Please enter student's age. ")
gradeLevel = input("Please enter student's grade level. ")

def insertLog(input1, input2, input3):
	sql = f"INSERT INTO students (name, age, gradeLevel) VALUES ('{input1}', '{input2}', '{input3}')"
	cur.execute(sql)
insertLog(name,age,gradeLevel)


cur.close()
db.close()

File: sqlINSERT.php
<?php

$servername = "localhost";
$username = "user";
$password = "password";
$dbname = "school";
$name = 'logan';
$age = 17;
$gradeLevel = 12;

$conn = new mysqli($servername, $username, $password, $dbname);

if($conn->connect_error){
    die("connection failed:".$conn->connect_error);
}

$sql = "INSERT INTO students (name, age, gradeLevel) VALUES ('$name', '$age', '$gradeLevel')";

if ($conn->query($sql) === TRUE) {
    echo "New record created successfully";
} else {
    echo "Error: " . $sql . "<br>" . $conn->error;
}

$conn->close();
?>
